source/includes/_messages.md

```diff
@@ -48,6 +49,7 @@ bcc   | array of `Recipient` objects (see below)
 subject | email subject line
 savedBy | when starred, this is the `Account._id`
 inbox | email address to which this message belongs
+originalInbox | email address which originally received the message
```

Comment:
...this will only be different if the message was forwarded, or rerouted to a catch-all inbox.

Reply:
added additional information 5a62c8c